carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/AbstractCapabilities.java

```diff
@@ -64,7 +68,10 @@ protected DesiredCapabilities initBaseCapabilities(DesiredCapabilities capabilit
         if ("*".equalsIgnoreCase(browserVersion)) {
             browserVersion = "";
         }
-        capabilities.setVersion(browserVersion);
+        if (!browserVersion.isEmpty()) {
+            capabilities.setCapability(Parameter.BROWSER_VERSION.getKey(), browserVersion);
```

Comment:
remove this line:
capabilities.setCapability(Parameter.BROWSER_VERSION.getKey(), browserVersion);